### ISSUE TITLE
Any way to have "generic" backup locations?

### ISSUE THREAD

**Issue by FlailAway** [bit-team/backintime#699]
Ubuntu 14.04-LTS, i5 laptop.
Hi, Love BiT, but just discovered that I don't think I can copy a backup set to another drive.

I have two encrypted USB-drives and they Mount as "ecDrive1" and "ecDrive2" I have two BiT-Profiles and make backups to the two drives one after the other.

I just created a new full backup folder and it took about 20-minutes to make the new backup. I thought I would copy that backup to the "ecDrive2" drive, but the hard-paths in it show "ecDrive1" so I assume it would not Restore from that.

Is there some way to have a Generic Path to the backup set for a Restore? I cannot change the drive assignments "ecDrive1" or "ecDrive2," as the encryption software defines that when originally encrypting the drive.

**Comment by Germar** [comment]
Not sure I understood your request. But of course you can copy a backup set or change the path. You only need to change the path `Where to save snapshots` and maybe the Host/User/ProfileID in Settings.

Just remember to use a [copy method which preserve hard-links](https://github.com/bit-team/backintime/wiki/FAQ#how-to-move-snapshots-to-a-new-hard-drive)

**Comment by FlailAway** [comment]
OK, thanks, I get it now. It was a missing Folder thing but I wasn't getting any error message when the Backup was on a new drive and the restore-to-folder ("myRouter") did not exist in the Home folder. 

Then BiT did nothing when I clicked "Restore," so I manually created an empty restore-to-folder ("Home/myRouter") and it worked OK. It seems that it cannot Restore a Folder if it doesn't yet exist at the destination. Once it had a correct Folder-name, it was happy to then fill it with the backed up data.
